src/lib/Previews.svelte

```diff
@@ -70,7 +71,7 @@
 
 <section id="example">
 	{#each Object.keys(widths) as width}
-		<div class="size">
+		<div class="size {width === '100%' && 'full-width'}">
```

Comment:
In Svelte, this is more semantically achieved by the [`class:` directive](https://svelte.dev/examples/classes)

```svelte
<div class="size" class:full-width={width === 100%}>
```